src/ui/generic/qrcodeScanner.tsx

```diff
@@ -90,7 +90,7 @@ const mapDispatchToProps = (dispatch: ThunkDispatch) => ({
 
     return handler
       ? dispatch(
-          withLoading(toggleLoading)(
+          withLoading(setDeepLinkLoading)(
```

Comment:
Is it correct to be using setDeepLinkLoading here? Is it the same as the comment from claims - that we need to use the NavigationContainer to show the spinner?